packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -272,14 +272,16 @@ export class SummaryGenerator {
                         generateTelemetryProps = {
                             ...generateTelemetryProps,
                             clientSequenceNumber: summaryData.clientSequenceNumber,
-                            submitOpDuration: summaryData.submitOpDuration,
                         };
                     }
                 }
             }
 
-            logger.sendTelemetryEvent({ eventName: "GenerateSummary", ...generateTelemetryProps });
-            if (summaryData.stage !== "submit") {
+            if (summaryData.stage === "submit") {
+                logger.sendTelemetryEvent({ eventName: "GenerateSummary", ...generateTelemetryProps });
+            }
+            else {
+                // Log event here on summary success only, as Summarize_cancel duplicates failure logging.
```

Comment:
I'm confused about this comment. Is it referring to the next line? It doesn't seem to be logging an event (except maybe as a side effect).